themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md

```diff
@@ -0,0 +1,153 @@
+>
+> Here's a Pulumi TypeScript program to deploy a static website on Azure using the `azure-static-website.Website` resource.
+>
+> ```ts
+> import * as pulumi from "@pulumi/pulumi";
+> import * as azure from "@pulumi/azure";
+> import * as staticWebsites from "@pulumi/azure-static-website";
+>
+> const website = new staticWebsites.Website("website", {
+>   sitePath: websitePath,
+>   resourceGroupName: resourceGroup.name,
+>   withCDN: true,
+> });
+> ```
+
+### New products
+
+When asked for modern resources such as [Google AlloyDB](https://cloud.google.com/alloydb), launched six weeks ago, Pulumi AI now references Registry documentation to answer "google alloydb with an automatic backup policy":
+
+> Based on the Pulumi Registry results and trying to accomplish your goal of creating an AlloyDB cluster with an automatic backup policy, we can create a Google Cloud Platform (GCP) AlloyDB cluster using the [`gcp.alloydb.Cluster`](/registry/packages/gcp/api-docs/alloydb/cluster/) resource. You can set the `automatedBackupPolicy` property with the necessary configuration to enable automatic backups.
+>
+> Here's a Pulumi program in TypeScript that creates an AlloyDB Cluster with an automatic backup policy:
+>
+> ```typescript
+> import * as pulumi from "@pulumi/pulumi";
+> import * as gcp from "@pulumi/gcp";
+>
+> const projectId = gcp.config.project;
+> const location = "us-central1";
+>
+> const network = new gcp.compute.Network("alloydb-network", {
+>   autoCreateSubnetworks: false,
+> });
+>
+> const alloydbCluster = new gcp.alloydb.Cluster("alloydbCluster", {
+>   clusterId: "my-alloydb-cluster",
+>   location,
+>   project: projectId,
+>   network: network.selfLink,
+>   automatedBackupPolicy: {
+>     enabled: true,
+>     labels: {
+>       "backup-type": "automated",
+>     },
+>     location: "us-central1",
+>     quantityBasedRetention: {
+>       count: 7,
+>     },
+>     timeBasedRetention: {
+>       retentionPeriod: "7d",
+>     },
+>     weeklySchedule: {
+>       daysOfWeeks: ["MONDAY", "WEDNESDAY", "FRIDAY"],
+>       startTimes: [
+>         {startTime: "04:00:00"},
+>       ],
+>     },
+>   },
+> });
+>
+> export const alloyDBClusterName = alloydbCluster.name;
+> ```
+
+## Getting Started with the Enhanced Pulumi AI
+
+Our enhancements to Pulumi AI have made it a more useful tool for generating accurate and relevant cloud infrastructure solutions. We're excited about the future of Pulumi AI as we continue to integrate it into [Pulumi Cloud](https://app.pulumi.com) and empower everyone to become a cloud engineer.
+
+These enhancements are live now at [Pulumi AI](/ai/), and we've added links to Pulumi AI in pages on the Pulumi Registry, for example on the [`eks.Cluster`](/registry/packages/eks/api-docs/cluster/) docs mentioned earlier.
+
+Give them a try and share your experiences with us. Together, let's build a future where infrastructure as code becomes more accessible, efficient, and enjoyable for everyone.
```

Comment:
Should we mention that we also added the ability to share your conversation results with others?  Perhaps include a screenshot like this:

<img width="496" alt="Screenshot 2023-05-17 at 9 53 28 PM" src="https://github.com/pulumi/pulumi-hugo/assets/223467/cfe5c070-1555-471e-b6c4-a66a6df0590c">

Reply:
Added